src/tests.rs

```diff
@@ -59,6 +60,22 @@ fn parse_file() {
 }
 
 #[test]
+fn read_write_file() {
+    let mut path = Path::new(env!("CARGO_MANIFEST_DIR")).to_path_buf();
```

Comment:
Neat, I didn't know about tempdir!

Two quick questions. Is it ok to put `extern crate tempdir` into `tests.rs`, or should it be behind config flag in `lib.rs`? And second, is it ok to say `TempDir::new("test_rm_tempdir").unwrap()`?